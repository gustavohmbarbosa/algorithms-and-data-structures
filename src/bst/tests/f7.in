7
66
1
50
1
2
1
-2
1
20
1
10
1
30
1
40
1
15
1
31
7
15
7
31
7
50
7
2
7
60
99