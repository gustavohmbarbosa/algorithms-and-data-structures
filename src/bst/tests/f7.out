-1
10
30
40
-2
-1
